packages/SwingSet/test/virtualObjects/test-representatives.js

```diff
@@ -379,46 +379,46 @@ test('virtual object gc', async t => {
     capargs({ '@qclass': 'undefined' }),
   );
   const remainingVOs = {};
-  for (const key of hostStorage.kvStore.getKeys('v1.vs.', 'v1.vs/')) {
+  for (const key of hostStorage.kvStore.getKeys('v2.vs.', 'v2.vs/')) {
```

Comment:
What do you think about factoring that string out, to make this sort of change easier in the future? I guess it would make this delta even bigger, with all the single-quotes being turned into backquotes, for eg
```js
`${v}.vs.baggageID`: 'o+6/1',
```

Reply:
I'm inclined to agree, given the tedium of these kinds of edits.  If nothing else, Murphy's Law says that if we make it easier to update if the ordering changes in the future, then we ensure that such changes never happen.  Yay for stability!